packages/cli/src/index.ts

```diff
@@ -32,6 +34,10 @@ program
   .command('createapp <name>')
   .description('Creates a new directory with a new FoalTS app.')
   .action((name: string) => {
+    if (args.length > 1) {
+      console.log(chalk.red.bold('\n Kindly provide only one argument as the project name'));
+      process.exit(1);
```

Comment:
Using `process.exit` after a `console.log` may truncate the printed data in some situations (see https://nodejs.org/api/process.html#process_process_exit_code and https://nodejs.org/api/process.html#process_a_note_on_process_i_o). Could we use a `return;` instead?

Reply:
👍